releasenotes/index.md

```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [2.1.0](/releasenotes/2.1.0) | [Javadoc](/reference/javadoc/2.1.x/)
```

Comment:
This link seems to be broken:

http://jclouds.apache.org/reference/javadoc/2.1.x/

Is that expected?

Reply:
yes @demobox I think I've not executed that script. When should I do that?

Reply:
The steps are [here](https://cwiki.apache.org/confluence/display/JCLOUDS/Publishing+Javadoc), from what I can make out - listed as step 19 in the [release guide](https://cwiki.apache.org/confluence/display/JCLOUDS/Releasing+jclouds), I think?

Reply:
@demobox I think you are right, I missed step 19

I think we need to update the notes at https://cwiki.apache.org/confluence/display/JCLOUDS/Publishing+Javadoc as they are not working for me. I've tried to tweak the script - see new commit in this PR - but I'm afraid I don't know enough of the process to fix it properly. I don't get the svn part, in particular

Reply:
> but I'm afraid I don't know enough of the process to fix it properly

@nacx I think you've run this most recently...any thoughts?

> I don't get the svn section at line 55 shows me of course

From what I can see in [`deploy-site.sh`](https://github.com/jclouds/jclouds-site/blob/master/deploy-site.sh#L21-L25), I think the assumption is that that is run first. That would make the directory into which `jclouds-site` is cloned both a Git and an SVN repository, and then the [`svn status` command](https://github.com/jclouds/jclouds-site/blob/master/deploy-javadoc.sh#L54) in `deploy-javadoc.sh` should work..?

Reply:
Javadocs are not pushed to the staging site; we just push them once the page is live, so I'd say there's no issue here.